File: Data Structures & Algorithms/sliding-window-maximum/submission-0.java
class Solution {
    public int[] maxSlidingWindow(int[] nums, int k) {
        // Brute force approach
        // Let go over each windown
        int[] res = new int[nums.length - k + 1];
        for (int i=0; i<=nums.length - k; i++) {
            int maxNum = nums[i];
            for (int j = i; j<i+k; j++) {
                maxNum = Math.max(maxNum, nums[j]);
            }
            res[i] = maxNum;
        }
        return res;
    }
}

File: Data Structures & Algorithms/daily-temperatures/submission-0.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        
        # Brute Force
        res = [0] * len(temperatures)
        for i in range(len(temperatures) - 1):
            day = 0
            for j in range(i+1, len(temperatures)):
                day += 1
                if temperatures[j] > temperatures[i]:
                    res[i] = day
                    break

        return res